calicoctl/commands/node/status.go

```diff
@@ -69,26 +72,35 @@ Description:
 
 	fmt.Printf("Calico process is running.\n")
 
+	if !psContains("bird", processes) && !psContains("bird6", processes) && !psContains("calico-bgp-daemon", processes) {
```

Comment:
I think this logic should be something like this: 
```go
	if psContains("bird", processes) || psContains("bird6", processes) {
		if psContains("bird", processes) {
			printBGPPeers("4")
		} else {
			fmt.Printf("\nINFO: BIRDv4 process: 'bird' is not running.\n")
		}
		if psContains("bird6", processes) {
			printBGPPeers("6")
		} else {
			fmt.Printf("\nINFO: BIRDv6 process: 'bird6' is not running.\n")
		}
	} else if psContains("calico-bgp-daemon", processes) {
		printGoBGPPeers("4")
		printGoBGPPeers("6")
	} else {
		fmt.Printf("\nNone of the BGP backend processes (BIRD or GoBGP) are running.\n")
	}
```

The reason for checking if one of the bird processes are running is because someone can have Calico running only in IPv4 or v6 mode, so they might turn off the other process.